js/components/dataset/resource/file-form.vue

```diff
@@ -57,10 +57,12 @@
 import API from 'api';
 import Dataset from 'models/dataset';
 import CommunityResource from 'models/communityresource';
+import FormHorizontal from 'components/form/horizontal-form.vue';
```

Comment:
I suppose you're trying not to redo the templates but `HorintalForm` would have made more sense.